.github/workflows/snorkell-auto-documentation.yml

```
@@ -0,0 +1,19 @@
+name: Penify - Revolutionizing Documentation on GitHub
+
+on:
+  push:
+    branches: ["master"]
+  workflow_dispatch:
+
+jobs:
+  Documentation:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Penify DocGen Client
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)